apps/desktop/tests/views/SettingsView/providerConfig.test.ts

```
@@ -0,0 +1,245 @@
+                provider: createProvider({ api_key: 'ta_live_abc' }),
+            },
+        });
+
+        await flushPromises();
+
+        expect(wrapper.get('[data-testid="settings-managed-activity-status"]').text()).toContain(
+            'octocat'
+        );
+        expect(wrapper.get('[data-testid="settings-managed-activity-page-button"]').text()).not.toBe(
+            ''
+        );
```

ESLint

[error] Replace `wrapper.get('[data-testid="settings-managed-activity-page-button"]').text()).not.toBe(⏎············''⏎········` with `⏎············wrapper.get('[data-testid="settings-managed-activity-page-button"]').text()⏎········).not.toBe(''`

(prettier/prettier)